lang/python/python-cryptodome/patches/001-fix-libgmp-loading.patch

```diff
@@ -0,0 +1,11 @@
+--- a/lib/Crypto/Math/_IntegerGMP.py
++++ b/lib/Crypto/Math/_IntegerGMP.py
+@@ -92,7 +92,7 @@ gmp_defs = """typedef unsigned long UNIX_ULONG;
+         int __gmpz_divisible_ui_p (const mpz_t n, UNIX_ULONG d);
+         """
+ 
+-lib = load_lib("gmp", gmp_defs)
++lib = load_lib("libgmp.so.10", gmp_defs)
```

Comment:
would `libgmp.so` work as well ?
would it make sense to use that ?
i haven't checked if there is `libgmp.so.20` or something like that ;

but if it makes sense [and it is possible] i would recommend using the more generic [without version library name]

Reply:
No. libgmp package does not contain `libgmp.so`.
```
# opkg files libgmp
Package libgmp (6.1.2-1) is installed on root and has the following files:
/usr/lib/libgmp.so.10.3.2
/usr/lib/libgmp.so.10
```
```
>>> import Crypto.Math._IntegerGMP
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/site-packages/Crypto/Math/_IntegerGMP.py", line 95, in <module>
    lib = load_lib("libgmp.so", gmp_defs)
  File "/_raw_api.py", line 181, in load_lib
  File "/__init__.py", line 348, in __init__
OSError: Error loading shared library libgmp.so: No such file or directory
```

Reply:
ack
thanks for clarifying;
let's leave `lib = load_lib("libgmp.so.10", gmp_defs)` & report it upstream to pycryptodome to fix this on their side